test/indexer/test_indexer_controller.py

```diff
@@ -89,12 +90,41 @@ def test_invalid_notification(self):
         self.assertRaises(AssertionError, self.controller.contribute, event)
 
     def test_remote_reindex(self):
-        event = [self._mock_sqs_record(action='reindex', prefix='ff')]
-        with mock.patch.object(target=AzulClient,
-                               attribute=AzulClient.remote_reindex_partition.__name__) as mock_method:
-            mock_method.return_value = True
-            self.controller.contribute(event)
-            mock_method.assert_called_once_with(dict(action='reindex', prefix='ff'))
+        mock_ppl = mock.patch.dict(os.environ, AZUL_PARTITION_PREFIX_LENGTH='0')
+        mock_dqp = mock.patch.dict(os.environ, AZUL_DSS_QUERY_PREFIX='ff')
+        mock_dss = mock.patch.dict(os.environ, AZUL_DSS_ENDPOINT='foo_source')
+        with mock_ppl, mock_dqp, mock_dss:
```

Comment:
Why did this get so much longer? Is this a remnant from another PR?

Reply:
It's been a while since we had this discussion but if i recall correctly; this test modification came from a previous discussion about having a `partition_prefix_length` of 0. We wanted to ensure that the synthesized notification work as expected. Rather than creating a new test that had this functionality we opted to extend the `test_remote_reindex`.